CHANGELOG.md

```diff
@@ -1,14 +1,22 @@
 # Change Log
 
-## [2.1.0](https://github.com/dingwilson/SwiftVideoBackground/tree/2.1.0)
```

Comment:
I think you accidentally deleted 2.1.0 😂

Reply:
Damnit lol